tests/specs/scss/functions.scss

```diff
@@ -96,6 +94,19 @@
     $get: config-has("test-not-found");
     $expect: false;
 
-    @include assert-equal($get, $expect, "Returns true.");
+    @include assert-equal($get, $expect, "Returns false.");
+  }
+
+  // Testing set functions
+  @include test("config-reset [function] Remove config map.") {
+    @include config-set("remove.bar", "foo");
+    @include config-set("remove.color", #fff);
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.